src/Symfony/Component/ClassLoader/ClassMapGenerator.php

```diff
@@ -134,7 +134,7 @@ private static function findClasses($path)
                     }
 
                     if ($isClassConstant) {
-                        continue;
+                        continue 2;
```

Comment:
I suggest using just `break` here and also removing the `default` case which looks superfluous, isn't it?